.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -119,6 +126,7 @@ stages:
       artifact: build-linux-$(matrixName)-working-directory
 
 - stage: build_macos
+  condition: ne(variables['Build.Reason'], 'Schedule') # skip for scheduled builds
```

Comment:
I would be nice to have more generic variables. I imagine that we will want to manually run Crank at some point

Reply:
Added this option in https://github.com/DataDog/dd-trace-dotnet/pull/1515/commits/c9e9cb54e7a3b16234180611280bd728563f1335, so you can set the `run_benchmarks_only` variable when triggering the pipeline manually. Can also set the `skip_benchmarks` variable for the opposite effect
![image](https://user-images.githubusercontent.com/18755388/120793446-ce957180-c52e-11eb-93bc-f5a00285b089.png)